dashboard/pull_requests/pull_requests_template.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
```

Comment:
Could you put the templates in the same documentation? And and with `.template`, like the other does.